retrievaltool/src/main/java/org/duracloud/retrieval/mgmt/RetrievalWorker.java

```diff
@@ -121,6 +122,13 @@ public void run() {
                 props = retrieveToFile(localFile, listener);
                 succeed(localFile.getAbsolutePath());
             }
+        } catch (MissingContentException mce) {
+            logger.error("The remote file " +
```

Comment:
This log message is not needed given that the following call to `missing()` immediately logs the same information.